### ISSUE TITLE
Support Node.js v18 by Updating Peer restify Dependency to Permit Version 10.0.0 or newer

### ISSUE THREAD

**Issue by sternam** [DatatracCorporation/restify-cors-middleware2#21]
For Node v18, `restify` must be at least version `10.0.0` causing the following npm error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: restify-cors-middleware2@2.1.2
npm ERR! Found: restify@9.0.0
npm ERR! node_modules/restify
npm ERR!   restify@"9.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer restify@"2.6.x - 8.x.x" from restify-cors-middleware2@2.1.2
npm ERR! node_modules/restify-cors-middleware2
npm ERR!   restify-cors-middleware2@"2.1.2" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: restify@8.6.1
npm ERR! node_modules/restify
npm ERR!   peer restify@"2.6.x - 8.x.x" from restify-cors-middleware2@2.1.2
npm ERR!   node_modules/restify-cors-middleware2
npm ERR!     restify-cors-middleware2@"2.1.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/astern/.npm/_logs/2022-11-18T16_40_23_496Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/astern/.npm/_logs/2022-11-18T16_40_23_496Z-debug-0.log
```

**Comment by JimPatterson** [comment]
Yep, looks like it is time for refresh of all the dependancies,  I don't have time today, but after the US holidays, I'll work on that.

**Comment by sternam** [comment]
@JimPatterson Thanks!  Turns out restify 9.0.0 still does not support Node.js v18 yet, but I see an open PR over there, once it's done the next release will be the one we need for the tests to pass with v18: https://github.com/restify/node-restify/pull/1929

**Comment by sternam** [comment]
@JimPatterson Ready to start this one now since restify 10.0.0 is out and supports Node.js v18

**Comment by sternam** [comment]
@JimPatterson I've taken the liberty of updating just the restify peer and dev dependencies, and regenerating the package-lock.json file to get all this repo's tests passing with Node.js v18 (they were failing before because restify would fail upon startup before version 10.0.0 on Node.js v18).  The only thing I didn't do in my PR is bump the version number because I wanted to let you decide if it is a minor or major increase.  https://github.com/DatatracCorporation/restify-cors-middleware2/pull/22

**Comment by JimPatterson** [comment]
This has been released as 2.2.0.  Thanks for this.
